i18n/OrkamvDataApiPlugin_de.ts

```diff
@@ -111,14 +125,42 @@
         <translation>Zielverzeichnis:</translation>
     </message>
     <message>
-        <location filename="../orkamv_data_api_plugin_dialog_base.ui" line="135"/>
+        <location filename="../orkamv_data_api_plugin_dialog_base.ui" line="142"/>
         <source>All needed files will be stored in the target directory.</source>
         <translation>Im Zielverzeichnis werden alle benötigten Daten abgelegt.</translation>
     </message>
     <message>
-        <location filename="../orkamv_data_api_plugin_dialog_base.ui" line="168"/>
+        <location filename="../orkamv_data_api_plugin_dialog_base.ui" line="149"/>
+        <source>Layer selection:</source>
+        <translation>Layerauswahl:</translation>
+    </message>
+    <message>
+        <location filename="../orkamv_data_api_plugin_dialog_base.ui" line="186"/>
+        <source>All</source>
+        <translation>Alle</translation>
+    </message>
+    <message>
+        <location filename="../orkamv_data_api_plugin_dialog_base.ui" line="196"/>
+        <source>By Group</source>
+        <translation>nach Gruppe</translation>
```

Comment:
```suggestion
        <source>By group</source>
        <translation>Nach Gruppe</translation>
```

Reply:
solved in https://github.com/terrestris/orkamv-data-api-plugin/pull/13/commits/df45de6373a72276ad0f9d50054a2a28bb700fec